.github/workflows/windows-release.yml

```diff
@@ -215,9 +212,7 @@ jobs:
         path: _build/_CPack_Packages/win64/NSIS/NSISOutput.log
 
     - name: Upload NSIS installer
-      uses: actions/upload-release-asset@v1.0.2
+      uses: softprops/action-gh-release@v0.1.15
+      if: startsWith(github.ref, 'refs/tags/')
```

Comment:
This if is not required since this workflow is started only when a release is created. It always evaluates to `true` AFAIK